package.json

```diff
@@ -89,7 +89,7 @@
     "@rollup/plugin-babel": "^5.3.0",
     "@rollup/plugin-node-resolve": "^11.2.0",
     "concurrently": "^5.3.0",
-    "eslint": "^7.20.0",
+    "eslint": "^7.21.0",
```

Comment:
Please remove all changes to `package.json` and `package-lock.json`.

Reply:
Done! I however have no idea why he flags a change in the last rule of `package-lock.json`.